fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/CastToEmptyRule.java

```diff
@@ -39,6 +39,9 @@ public CastToEmptyRule() {
 
     @Override
     public boolean check(OptExpression input, OptimizerContext context) {
+        if (!(input.getOp() instanceof LogicalOperator)) {
+            return false;
+        }
```

Comment:
```suggestion

```

Reply:
physical operator will invoke this  rule， will be error

Reply:
should update?